README.md

```diff
@@ -76,6 +76,18 @@ To learn more about the platform, check out our [documentation](https://github.c
   ./prepare.sh
   ```
 
+- Using component manager
```

Comment:
There is nothing about using, just installation

Reply:
i mean installation method using component manager